.idea/runConfigurations/Unit_Tests.xml

```diff
@@ -0,0 +1,17 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Unit Tests" type="AndroidJUnit" factoryName="Android JUnit">
+    <module name="app" />
+    <useClassPathOnly />
+    <option name="ALTERNATIVE_JRE_PATH_ENABLED" value="true" />
+    <option name="ALTERNATIVE_JRE_PATH" value="/usr/lib/jvm/java-11" />
```

Comment:
This might help people to run the Unit Tests which require Java 11 while Android Studio otherwise uses Java 8.